### PR TITLE
Add JDK 11 and load install-jdk.sh on-the-fly

### PR DESCRIPTION
**Pull request by sormuras**
No more updates needed when new EA feature builds are published or paths to existing archives are changed.

### CODE REVIEW

**Comment by sormuras** [comment]
Allow JDK 11 builds to fail. `error-prone` is not there, yet?

```
bad class file: /modules/java.base/java/io/IOException.class
    class file has wrong version 55.0, should be 53.0
```

**Comment by sormuras** [comment]
With the GA of JDK 10 -- http://jdk.java.net/10/ -- the PR needs (a) to be modified a bit and (b) to be merged. There's no longer an "Oracle JDK 10 build"...

Regarding (a): I'm on it.

**Comment by sormuras** [comment]
@JakeWharton / @swankjesse -- next build on Travis CI is expected to fail, if this PR is not merged. `./.buildscript/install-jdk.sh` is outdated and replaced by this PR, i.e. `install-jdk.sh` loaded on-the-fly.

File: .travis.yml [hunk 1]
```diff
@@ -2,14 +2,28 @@ language: java
...
+  allow_failures:
+    # Oracle JDK is not yet available on Travis CI: https://github.com/travis-ci/travis-ci/issues/9368
+    - env: JDK='Oracle JDK 10'
+    # ErrorProne/javac is not yet working on JDK 11
+    - env: JDK='Oracle JDK 11'
+    - env: JDK='OpenJDK 11'
+
+before_install:
+  - unset _JAVA_OPTIONS
+  - wget https://raw.githubusercontent.com/sormuras/bach/master/install-jdk.sh
```
Comment (JakeWharton) [inline .travis.yml]: Can we pin this to a SHA rather than a branch?

Comment (sormuras) [reply .travis.yml]: You'll lose the auto-update feature/service I provide. Oracle/Java.net keep changing the URLs now and then...

The manual loading of `install-jdk.sh` might be superseded by the integration in Travis CI, soon. See https://github.com/travis-ci/travis-build/pull/1347 for details.

Comment (sormuras) [reply .travis.yml]: Just released `1.0.1`. So we may pin it to:

`https://github.com/sormuras/bach/raw/1.0.1/install-jdk.sh`

**Comment by sormuras** [comment]
@JakeWharton Force-pushed a pinned version and removed the outdated `install-jdk.sh` copy from `.buildscript`.

**Comment by sormuras** [comment]
@swankjesse All red crosses at Travis-CI hurt my testing guts... ❌ 
Any chance to make the builds happy again by merging this PR? ✔️

**Comment by swankjesse** [comment]
Thanks!

**Comment by sormuras** [comment]
You're welcome!

**Pull request merged**